bot.go

```diff
@@ -36,6 +36,18 @@ func New() {
 	<-sc
 	return
 }
+func getNumOptions() []string {
+	arr := []string{"1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "6️⃣", "7️⃣", "8️⃣", "9️⃣"}
+	return arr
+}
+func containAtIndexNum(s []string, e string) (int, bool) {
+	for i := 0; i < len(s); i++ {
+		if e == s[i] {
+			return i, true
+		}
+	}
+	return 0, false
```

Comment:
配列の各要素に対する操作（いわゆるforeach）はrangeを使って書けます

```suggestion
	for i, v := range s {
		if e == v {
			return i, true
		}
	}
	return 0, false
```

Reply:
ありがとうございます。
変更内容は上記で反映します。